rocAL/rocAL/source/augmentations/color_augmentations/node_color_temperature.cpp

```diff
@@ -21,43 +21,32 @@ THE SOFTWARE.
 */
 
 #include <vx_ext_rpp.h>
-#include <VX/vx_compatibility.h>
 #include "node_color_temperature.h"
 #include "exception.h"
 
 ColorTemperatureNode::ColorTemperatureNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
         Node(inputs, outputs),
-        _adj_value_param(ADJUSTMENT_RANGE[0], ADJUSTMENT_RANGE[1])
-{
-}
+        _adj_value_param(ADJUSTMENT_RANGE[0], ADJUSTMENT_RANGE[1]) { }
 
-void ColorTemperatureNode::create_node()
-{
+void ColorTemperatureNode::create_node() {
     if(_node)
         return;
 
-    _adj_value_param.create_array(_graph , VX_TYPE_INT32, _batch_size);
-
-    // _node = vxExtrppNode_ColorTemperaturebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _adj_value_param.default_array(), _batch_size);
+    _adj_value_param.create_array(_graph , VX_TYPE_UINT32, _batch_size);
+    _node = vxRppColorTemperature(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _adj_value_param.default_array(), _input_layout, _output_layout, _roi_type);
 
     vx_status status;
     if((status = vxGetStatus((vx_reference)_node)) != VX_SUCCESS)
-        THROW("Adding the color temp batch (vxExtrppNode_ColorTemperaturebatchPD) node failed: "+ TOSTR(status))
-
+        THROW("Adding the ColorTemperature (vxRppColorTemperature) node failed: "+ TOSTR(status))
```

Comment:
Please add space before +
Applicable too all such instances in similar warning messages